modules/webgl/src/context/context.js

```diff
@@ -171,15 +171,9 @@ export function resizeGLContext(gl, options = {}) {
   // Resize browser context
   if (gl.canvas) {
     const devicePixelRatio = getDevicePixelRatio(options.useDevicePixels);
-    setDevicePixelRatio(gl, devicePixelRatio, options);
+    setDevicePixelRatio(gl, devicePixelRatio);
     return;
   }
-
-  // Resize headless gl context
```

Comment:
@Pessimistress @ibgreen , any known use cases of code block?

Reply:
headless-gl support.

Reply:
ok, I will keep this for now.